src/Ordering.API/Infrastructure/OrderingContextSeed.cs

```diff
@@ -19,6 +18,8 @@ public async Task SeedAsync(OrderingContext context)
 
     private static IEnumerable<CardType> GetPredefinedCardTypes()
     {
-        return Enumeration.GetAll<CardType>();
+        yield return new CardType { Id = 1, Name = "Amex" };
```

Comment:
I considered removing `OrderingContextSeed` altogether, but kept it for sake of minimizing changes